docs/conf/australia/2019/news/cfp-reminder.rst

```diff
@@ -0,0 +1,42 @@
+:template: {{year}}/generic.html
+
+.. post:: August 7, 2019
+   :tags: australia-2019, website, cfp, tickets
+
+
+CFP Reminder & Tickets on Sale
+==============================
+
+G'day fellow documentarians!
+
+Here is the latest update about our Australian conference.
+Our `call for proposals <https://www.writethedocs.org/conf/australia/2019/cfp/>`_ ends at midnight August 9, that is *tomorrow*!
+
+`Tickets for the conference are still on sale <https://www.writethedocs.org/conf/australia/2019/tickets/>`_.
+
+Read on for more details on both!
+
+Submit your talk by end of the day tomorrow!
+--------------------------------------------
+
+As always, we're eagerly looking forward to `your submissions <https://www.writethedocs.org/conf/australia/2019/cfp/#submit-your-proposal>`_! We’ll be reviewing all the talk proposals this month, and will be contacting everyone who submitted a proposal by {{ cfp.notification }}.
+
+Get your ticket for the conference
+----------------------------------
+
+This year the Australian conference will be **in Sydney, November 14-15, 2019**. The `full event website <https://www.writethedocs.org/conf/australia/2019/>`_ is live, with some preliminary details, and we'll be filling in more as it gets closer. We're already excited to see what this year has in store!
+
+If you're excited too, go ahead and `register for your ticket now <https://www.writethedocs.org/conf/australia/2019/tickets/>`_!
+
+**Please make sure to register for your ticket before you book your travel!** We cannot be responsible for any non-refundable travel costs, and if we reach capacity at the venue before you register, we cannot guarantee a spot for you, even if you booked your travel.
+
+Support our community by sponsoring
+-----------------------------------
+
+Every year we are fortunate to have the support of like-minded organizations and communities, who believe in the collaborative and welcoming community spirit and help us make our events more awesome.
+
+If your company is interested in sponsoring one or more conferences, check out `our prospectus for details on the different packages <https://www.writethedocs.org/conf/australia/2019/sponsors/prospectus/>`_ and drop us a line at `sponsorship@writethedocs.org <mailto:sponsorship@writethedocs.org>`_
+
+Looking forward to seeing you all in Sydney`!
```

Comment:
We should thank our current sponsors here, would be good to just build a small template block for this.

Reply:
If we added an added on `date` to sponsors, we could display more accurate "current" sponsors.